hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/HddsTestUtils.java

```diff
@@ -736,7 +736,8 @@ public static ContainerReplica.ContainerReplicaBuilder getReplicaBuilder(
             .setDatanodeDetails(datanodeDetails)
             .setOriginNodeId(originNodeId).setSequenceId(sequenceId)
             .setBytesUsed(usedBytes)
-            .setKeyCount(keyCount);
+            .setKeyCount(keyCount)
```

Comment:
Should this be keyCount or blockCount ?

Reply:
This is in server-scm which doesn't use block count. Here existing test case is just to determine whether replica has some keys or not. If some keys are present then setEmpty as false.